cypress/TestCases/FireboltCertification/Advertising.feature

```diff
@@ -141,10 +141,10 @@ Feature: Advertising
          | Without scope type               | get advertisingId without scope type        |
          | Invalid advertisingId scope      | get advertisingId with invalid scope        |
 
-   @sdk @transport @requiresPlatformImplementation
-   Scenario: Advertising.onPolicyChanged - Positive Scenario: Clearing event listeners
-      When '3rd party app' registers for the 'advertising onPolicyChanged' event using the 'Firebolt' API
-      And I clear 'advertising onPolicyChanged event' listeners
-      And 1st party app invokes the 'Firebolt' API to 'set true for allowAppContentAdTargeting'
-      Then 'Firebolt' platform responds to '1st party app' with 'set true for allowAppContentAdTargeting'
-      And 'Firebolt' platform does not trigger event for 'onAdvertisingPolicyChanged'
\ No newline at end of file
+      @Advertising @coreSDK @sdk @transport @requiresPlatformImplementation
```

Comment:
check the latest dev, tags are not correct

Reply:
updated the tags